ShutdownTimer/ShutdownTimer.yml

```diff
@@ -0,0 +1,24 @@
+app-id: org.example.ShutdownTimer
```

Comment:
this app-id is invalid. Despite you confirming 
>  The domain used for the application ID is controlled by the application developer(s) and the application id guidelines are followed.

The checkbox are not a CAPTCHA.